Formula/d/dwarfs.rb

```diff
@@ -0,0 +1,54 @@
+require_relative "macfuse"
```

Comment:
This is not allowed in Homebrew/core

Reply:
This should be resolved soon with a real upstream URL: https://github.com/mhx/dwarfs/issues/214#issuecomment-2102757897